components/suivi-form/acteurs/actors-autocomplete-input.js

```diff
@@ -58,7 +58,7 @@ const ActorsAutocompleteInput = ({isRequired, inputValue, inputError, onValueCha
       results={foundEtablissements}
       isLoading={isLoading}
       errorMessage={errorMessage || inputError}
-      renderItem={item => `${item.nom_complet} - ${secteursActivites[item.section_activite_principale]}`}
+      renderItem={item => `${item.nom_complet} - ${secteursActivites[item.section_activite_principale] || 'N/A'}`}
```

Comment:
`N/A` est une abréviation pour dire "Not Applicable" ou "Not Available". Gardons "non renseignée" ou même rien du tout puisque visiblement ce champ peut ne pas être renseigné et donc ça n’a pas trop de sens.